node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java

```diff
@@ -655,8 +666,12 @@ private void updateCpuDeltas(ContainerStats.CpuStats cpuStats) {
             return deltaSystemUsage == 0 ? Double.NaN : (double) deltaContainerKernelUsage / deltaSystemUsage;
         }
 
-        Long getThrottledTime() {
-            return deltaSystemUsage == 0 ? null : deltaThrottledTime;
+        double getThrottledTime() {
+            return deltaSystemUsage == 0 ? Double.NaN : 60d * deltaThrottledPeriods / deltaThrottlingActivePeriods;
```

Comment:
So deltaThrottlingActivePeriods cannot be 0 unless deltaSystemUsage is 0 !?

It's not clear where 60d comes from. I assume "time" is measured in seconds when it's a double.

Reply:
Yes, it can be 0 if throttling is disabled, otherwise no.

Change to `deltaThrottlingActivePeriods == 0`